routes/studentGroups.js

```diff
@@ -54,16 +70,17 @@ router.get('/site/:siteId', async (req, res) => {
 // add a student group
 router.post('/', async (req, res) => {
   try {
-    const { year, cycle, masterTeacherId, siteId, meetingDay, meetingTime } = req.body;
+    const { name, year, cycle, masterTeacherId, siteId, meetingDay, meetingTime } = req.body;
     isNumeric(year, 'Year must be a Number');
     isNumeric(masterTeacherId, 'Master teacher Id must be a Number');
     isNumeric(siteId, 'Site Id must be a Number');
-
     const group = await pool.query(
-      `INSERT INTO student_group (year, cycle, master_teacher_id, site_id, meeting_day, meeting_time)
-        VALUES ($1, $2, $3, $4, $5, $6)
-        RETURNING *;`,
-      [year, cycle, masterTeacherId, siteId, meetingDay, meetingTime],
+      `INSERT INTO student_group (
+        name, year, cycle, master_teacher_id,
+        site_id, meeting_day, meeting_time)
+      VALUES ($1, $2, $3, $4, $5, $6, $7)
+      RETURNING *;`,
+      [name, year, cycle, masterTeacherId, siteId, meetingDay, meetingTime],
     );
 
     const addedGroup = await pool.query(getStudentGroups(false), [group.rows[0].group_id]);
```

Comment:
The POST route is failing the following error. I think this call should not have false since the parameter is a string. 

Also, why do we need an additional pool.query in the first place? Can't you just send group like you did in the post route for `students.js`?

<img width="637" alt="image" src="https://user-images.githubusercontent.com/57849152/162826767-8f2543c6-6df4-4ac1-a053-8b94c4eb479f.png">

Reply:
fixed.
the reason for having two queries is the first query is inserting the student group into the database. the second query is then used to retrieve that newly added row in the same format as all the other get requests in the `student-group` route by using the `getStudentGroups` method at the top of the page.